scene/3d/sprite_3d.h

```diff
@@ -209,22 +209,24 @@ class AnimatedSprite3D : public SpriteBase3D {
 	GDCLASS(AnimatedSprite3D, SpriteBase3D);
 
 	Ref<SpriteFrames> frames;
+	String autoplay;
+
 	bool playing = false;
-	bool playing_backwards = false;
-	bool backwards = false;
 	StringName animation = "default";
 	int frame = 0;
 	float speed_scale = 1.0;
+	float custom_speed_scale = 1.0;
 
 	bool centered = false;
 
-	bool is_over = false;
-	double timeout = 0.0;
+	real_t frame_speed_scale = 0.0;
```

Comment:
```suggestion
	real_t frame_speed_scale = 1.0;
```

To match the 2D counterpart.